examples/config/config

```diff
@@ -312,8 +320,8 @@ set history_disable_easter_egg 1
 @cbind  <Ctrl>p = hardcopy page
 
 # Web searching binds
-@cbind  gg<Google:>_         = uri http://www.google.com/search?q=\@<encodeURIComponent(%r)>\@
-@cbind  ddg<DuckDuckGo:>_    = uri http://duckduckgo.com/?q=%s
+# NOTE you may search Google by adding "!g" to your DuckDuckGo search string
+@cbind  d<DuckDuckGo:>_      = uri http://duckduckgo.com/?q=%s
```

Comment:
Search is probably the most used feature (of the entire internet?), so it should be as "close" as possible.

What do you mean with "nice"?